yb-voyager/src/dbzm/config.go

```diff
@@ -355,6 +355,9 @@ func (c *Config) String() string {
 
 func (c *Config) WriteToFile(filePath string) error {
 	config := c.String()
+	if c.Password == "" { //empty password have issues with Env variable https://yugabyte.atlassian.net/browse/DB-7533
+		config += fmt.Sprintf("\ndebezium.source.database.password=%s", c.Password)	
```

Comment:
Just asking: do you need quotes around the value to represent empty string?

Reply:
No, I don't think it is required. Earlier when env wasn't introduced and in the conf, we were setting like only. cc: @makalaaneesh